framework/player/init/acreSignalCalc.sqf

```diff
@@ -0,0 +1,84 @@
+// ACRE SIGNAL CALC by diwako ////////////////////////////////////////////////////////////////////////////
+/*
+    Enables custom signal strength setting mid mission.
+	Example:
+	Exec this globally in debug console to boost receive or send power times 2
+	There is still a max range as this just feeds new values into the original acre function
+
+	player setVariable ["acre_send_power",2,true];
+	player setVariable ["acre_receive_power",2,true];
+
+	Of course you can crank up that number as much as you like
+*/
+// INIT ///////////////////////////////////////////////////////////////////////////////////////////
+if(!hasInterface) exitWith {};
```

Comment:
I just see minor things with it, would move the ! outside of the brackets to fit how its done in the rest of the framework + maybe move your name as the maker from the title to the description (again to fit the rest of the framework)